decidim-admin/spec/system/admin_manages_newsletters_spec.rb

```diff
@@ -113,6 +113,13 @@
       visit decidim_admin.preview_newsletter_path(newsletter)
       expect(page).to have_content("Hello Sarah Kerrigan! Relevant content.")
     end
+
+    it "sends a test email to the logged user" do
+      visit decidim_admin.newsletter_path(newsletter)
+      click_link "Send me a test email"
```

Comment:
```suggestion
      perform_enqueue_jobs do
        click_link "Send me a test email"
      end
```
With this helper you will force rspec to run any `deliver_later` action before continue. This will enable to check if the email has been sent

Reply:
Thank you. It's done.